### DISCUSSION TITLE
Arcropolis has panicked

### DISCUSSION THREAD

**Discussion by Goldenzepp** [Raytwo/ARCropolis#524] [Issues]
When I tried to load smash, I get this error. It happens every time I load it.
![20230419_152310](https://user-images.githubusercontent.com/47874712/233191499-a393aff5-8d0b-4779-82ef-04a27ca3af80.jpg)

**Comment by Anastanat** [comment]
Hi. WHAT IS THIS?
![Uploading 20241022_182504.jpg…]()

**Comment by Anastanat** [comment]
![20241021_215138](https://github.com/user-attachments/assets/6d59c76b-5fd8-473d-b097-3a57932aa30c)
Help me!
